composer.json

```diff
@@ -65,7 +65,7 @@
         "tomasvotruba/cognitive-complexity": "^0.2",
         "tomasvotruba/lines": "^0.5",
         "tomasvotruba/type-coverage": "^0.2",
-        "tomasvotruba/unused-public": "^0.3",
+        "tomasvotruba/unused-public": "dev-main",
```

Comment:
using dev-main for now, because otherwise it crashes - see https://github.com/TomasVotruba/unused-public/pull/90